app/Helpers/database/game.php

```diff
@@ -68,6 +68,10 @@ function getGameMetadata(
         14 => "ORDER BY ach.Points DESC, ach.ID DESC ",
         5 => "ORDER BY ach.Title, ach.ID ASC ",
         15 => "ORDER BY ach.Title DESC, ach.ID DESC ",
+        6 => "ORDER BY 
+            CASE WHEN ach.type IS NULL THEN 2 WHEN ach.type = 'progression' THEN 0 WHEN ach.type = 'win_condition' THEN 1 ELSE 3 END, 
+            ach.ID ASC ",
```

Comment:
It would probably be better to use `ach.DisplayOrder` as the secondary sort, with `ach.ID` as the tertiary sort (i.e. use the "default=>" sort as the secondary sort).

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1857/commits/dc335a38c85df9a807b4276afa4966fac32151cc.